rotkehlchen/types.py

```diff
@@ -603,6 +603,13 @@ class AssetMovementCategory(DBEnumMixIn):
     WITHDRAWAL = 2
 
 
+class ExchangeAuthCredentials(NamedTuple):
+    api_key: Optional[ApiKey]
```

Comment:
can you add a doctring explaining where this used and if any of the fields need to be populated or can all be none